src/components/experimental/ComboBox/docs/ComboBox.stories.tsx

```diff
@@ -153,3 +154,81 @@ export const FullyControlled: Story = {
         );
     }
 };
+
+export const WithRef: StoryObj<typeof ComboBox> = {
+    render: () => {
+        const [items, setItems] = React.useState<Character[]>([]);
+        const [filterText, setFilterText] = React.useState('');
+        const comboBoxRef = useRef<HTMLInputElement>(null);
+
+        React.useEffect(() => {
+            let ignore = false;
+
+            async function startFetching() {
+                const res = await fetch(`https://swapi.py4e.com/api/people/?search=${filterText}`);
+                const json = await res.json();
+
+                if (!ignore) {
+                    setItems(json.results);
+                }
+            }
+
+            // eslint-disable-next-line no-void
+            void startFetching();
+
+            return () => {
+                ignore = true;
+            };
+        }, [filterText]);
+
+        const handleFocus = () => {
+            if (comboBoxRef.current) {
+                comboBoxRef.current.focus();
+                // eslint-disable-next-line no-console
+                console.log('input focused');
```

Comment:
do we need these logs?

Reply:
Hmm not really, in the case of focus and blur, the change in the input state is evident. As for `handleGetValue` maybe we can remove it completely, the focus and blur actions are demonstration enough probably.

Reply:
Oh this is for stories.. nevermind